lorawan-device/src/async_device/lora_radio.rs

```diff
@@ -86,16 +86,17 @@ where
         )?;
         let mut tx_pkt_params =
             self.lora.create_tx_packet_params(8, false, true, false, &mdltn_params)?;
-        let pw = match self.lora.get_board_type().into() {
-            ChipType::Sx1276 | ChipType::Sx1277 | ChipType::Sx1278 | ChipType::Sx1279 => {
-                if config.pw > DEFAULT_DBM {
-                    DEFAULT_DBM
-                } else {
-                    config.pw
-                }
-            }
-            _ => config.pw,
-        };
+        let pw = config.pw.min(DEFAULT_DBM);
+
+        #[cfg(feature = "defmt")]
+        if config.pw > DEFAULT_DBM {
+            defmt::warn!(
+                "Limiting requested tx power {} with region default: {}",
```

Comment:
Is it `region default` or `radio default max`?

Reply:
Is there a way to get region default from this location?

Reply:
I can't think of an easy way, or a way at all to be honest.